changelog.txt

```diff
@@ -1,4 +1,5 @@
 0.85.0
+ * BREAKING CHANGE: ChangeRealm#executeTransaction will catch a RuntimeException (no longer catches Error) and will log a useful error message, cancel the transaction and re-throw the exception when said RuntimeException occurs (#1682). 
```

Comment:
To much information I think.
Maybe something like  `BREAKING CHANGE: Realm.executeTransaction() now directly throws any RuntimeException instead of wrapping it in a RealmException (#1682)`